__tests__/app.test.js

```diff
@@ -117,7 +117,26 @@ it('should return an array of articles' , () => {
         })
       })
     })  
+  })
+
+  it('should return articles sorted by votes', () => {
+    return request(app)
+    .get('/api/articles?sort_by=votes&order=asc')
+    .expect(200)
+    .then(({ body }) => {
+     expect(body.articles).toBeSortedBy('votes', {descending: false})
+    })  
   }) 
+
+  it('should return articles sorted by author', () => {
+    return request(app)
+    .get('/api/articles?sort_by=author&order=desc')
+    .expect(200)
+    .then(({ body }) => {
+     expect(body.articles).toBeSortedBy('author', {descending: true})
+    })  
+  }) 
```

Comment:
This is a good start, but there are a few more missing tests that should be added for this endpoint:

A test to individually test each query works on its own:
E.g `/api/articles?sort_by=votes` and `/api/articles?order=asc` as separate tests

A test to ensure when given an invalid sort by an appropriate error is given back:
E.g. `/api/articles?sort_by=banana`

A test to ensure when given an invalid order an appropriate error is given back:
E.g. `/api/articles?order=banana`